x/smart/stream_dialer.go

```diff
@@ -94,10 +95,16 @@ type dnsEntryConfig struct {
 	TCP    *tcpEntryConfig   `yaml:"tcp,omitempty"`
 }
 
+type fallbackEntryConfig struct {
+	ConfigURL 	string 					`yaml:"configUrl,omitempty"`
+	// Don't verify the psiphon config format here, just pass it forward
+	Psiphon 	map[string]interface{} 	`yaml:"psiphon,omitempty"`
+}
+
 type configConfig struct {
-	DNS      []dnsEntryConfig `yaml:"dns,omitempty"`
-	TLS      []string         `yaml:"tls,omitempty"`
-	FALLBACK []string         `yaml:"fallback,omitempty"`
+	DNS      []dnsEntryConfig 		`yaml:"dns,omitempty"`
+	TLS      []string         		`yaml:"tls,omitempty"`
+	Fallback []fallbackEntryConfig 	`yaml:"fallback,omitempty"`
```

Comment:
This should be `any` in order to allow for the type switch.

Reply:
done